### PR TITLE
LPD-37344 Adapt jest tests

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-page-management** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:page-management-playwright

**Comment by sandrodw3** [comment]
ci:test:page-management-playwright

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/TopperItemActions.js [hunk 1]
```diff
@@ -59,6 +61,25 @@ export default function TopperItemActions({disabled, item}) {
 
 	const [openSaveModal, setOpenSaveModal] = useState(false);
 
+	const widget = useMemo(() => {
+		const fragmentEntryLink =
+			fragmentEntryLinks[item.config.fragmentEntryLinkId];
+
+		if (!fragmentEntryLink) {
+			return;
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/TopperItemActions.js:69]: I'd return the same thing in both returns :P

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/TopperItemActions.js [hunk 2]
```diff
@@ -59,6 +61,25 @@ export default function TopperItemActions({disabled, item}) {
...
+		const {instanceId, portletId} = fragmentEntryLink.editableValues;
+
+		if (!portletId) {
+			return null;
+		}
+
+		const widget = getWidget(getWidgets(), portletId);
+
+		return {...widget, portletId: getPortletId({instanceId, portletId})};
+	}, [fragmentEntryLinks, getWidgets, item]);
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/TopperItemActions.js:81]: We only need the portletId here right? the rest of the object is not used, so we may just return that

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/TopperItemActions.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/TopperItemActions.js:69]: True 😛

**Comment by sandrodw3** [comment]
ci:forward:force

**Comment by sandrodw3** [comment]
ci:forward:force

**Pull request closed without merging**